### PR TITLE
Fix/io event loops

### PR DESCRIPTION
**Pull request by rgruyters**
Fixes issue #57

### CODE REVIEW

**Comment by xirixiz** [comment]
Super! Eindelijk wat hulp, dat kan ik enorm waarderen! Thanks!

File: custom_components/afvalwijzer/sensor.py [hunk 1]
```diff
@@ -156,7 +166,16 @@ def update(self):
         count_today = self._config.get(CONST_COUNT_TODAY)
 
         try:
-            afvaldienst = Afvaldienst(provider, zipcode, housenumber, suffix, count_today)
+            afvaldienst = await self.hass.async_add_executor_job(
```
Comment (rgruyters) [inline custom_components/afvalwijzer/sensor.py:169]:
```
File "/config/custom_components/afvalwijzer/sensor.py", line 169
    afvaldienst = await self.hass.async_add_executor_job(
                 ^
SyntaxError: 'await' outside async function
```

Comment (rgruyters) [reply custom_components/afvalwijzer/sensor.py:169]: I don't think I can fix this is a quick way. https://developers.home-assistant.io/docs/integration_fetching_data#polling-api-endpoints

File: custom_components/afvalwijzer/sensor.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
```
Comment (brefra) [inline custom_components/afvalwijzer/sensor.py:7]: You are missing `asyncio `to be able to  use `wait`
```suggestion
import asyncio
```

File: custom_components/afvalwijzer/sensor.py (hunk 1 above)
Comment (brefra) [reply custom_components/afvalwijzer/sensor.py:169]: Remove this change and make the change at L129 
```self._fetch_trash_data.update()```
with
```await self._hass.async_add_executor_job(self._fetch_trash_data.update)```

**Comment by xirixiz** [comment]
I've updated your branch with the modifications of @brefra. Can you verify if the component is working properly now?

**Comment by rgruyters** [comment]
@xirixiz Tested it and all looks fine by me.

**Pull request merged**